server/core/runtime/runtime.go

```diff
@@ -48,8 +48,10 @@ type AsyncTFExec interface {
 
 // StatusUpdater brings the interface from CommitStatusUpdater into this package
 // without causing circular imports.
+//
+//go:generate pegomock generate -m --package mocks -o mocks/mock_status_updater.go StatusUpdater
 type StatusUpdater interface {
```

Comment:
Are there tests that we can add to runetime_test.go ?

Reply:
StatusUpdater is just interface and this impl is `CommitStatusUpdater`. This test is implemented in `commit_status_updater_test.go`, so not needed 👌 

> StatusUpdater brings the interface from CommitStatusUpdater into this package without causing circular imports.

<img width="1011" alt="image" src="https://user-images.githubusercontent.com/915731/212488834-e93d3339-4be8-494b-b122-0536630d5427.png">